server/utils/cloudFunctions/functions/countAdvertisersCloudFunctions/countAdvertisersConfig.js

```diff
@@ -12,6 +12,10 @@ const DEV_AGGREGATES_COLLECTION_NAME = "dev_aggregates";
 const DEV_AGGREGATES_COLLECTION = 
     DB.collection(DEV_AGGREGATES_COLLECTION_NAME);
 
+const PROD_AGGREGATES_COLLECTION_NAME = "aggregates";
+const PROD_AGGREGATES_COLLECTION = 
```

Comment:
nit: have this one on the same line maybe?

Reply:
80 char limit